### ISSUE TITLE
Have a selection of sounds for caps

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#931]
**Reported by briang1 on 2010-09-23 09:00**
At present, the sound that occurs when caps are encountered in cursor, review and typing is a very high frequency tone. I fully realise this has to be done this way as playing a wav would slow the process. However, on letting a friend hear this, he could hardly hear it. So,if some other type of sound could be made an option for people who either prefer lower frequencies... it is a bit piercing at the moment, and for those with hf hearing loss, it would be good.

**Comment by leonardder** [comment]
CC @Brian1Gaff

I think this is a valid request. We could also consider lowering the pitch of the beep a bit?

**Comment by bhavyashah** [comment]
A few thoughts:
* This ticket deals with the beep that is played whenever capitalization is encountered when the Beep for capitals check box part of the Voice Settings dialog is checked.
* It is highly unlikely that a proper .wav sound scheme will be introduced since this may cause performance slowdowns as noted by @brian1gaff in the ticket body.
* It is highly unlikely that the current beep will be revised keeping in mind of our lack of success in revamping other NVDA audio cues despite investment of significant time and resources.
* Therefore, what we could do to address this request is (a) reduce the volume of the beep and/or (b) reduce the pitch of the beep.
* Another ticket already discusses the implementation of an audio manager which would allow for the customization of the volume of NVDA sounds. If that ticket is broadened so as to encompass tones and beeps, then (a) of the previous point gets effectively resolved. If we further extend that other ticket's scope to include setting a custom pitch for these tones and beeps (and maybe NVDA sounds also), then (b) of the previous point also gets effectively resolved.
* Still, although an extensive audio manager would effectively resolve this ticket, the default state of the capitalization indicating beep requires to be changed. @leonardder mentions in https://github.com/nvaccess/nvda/issues/931#issuecomment-315991319 that reduction of the beep's pitch should suffice. From a earphones user's standpoint, the beep's volume might also need some reduction.
* In conclusion, whether the volume or the pitch (or both) of the beep requires adjustment is up to debate, because the audio manager should only serve as a supplement, IMHO.

**Comment by bhavyashah** [comment]
Please note that the 'other ticket' concerning an 'audio manager' as I mentioned in https://github.com/nvaccess/nvda/issues/931#issuecomment-324565820 is #1409.

**Comment by josephsl** [comment]
Ping @Brian1Gaff

**Comment by Adriani90** [comment]
@josephsl what info do you need?

**Comment by josephsl** [comment]
Hi,

In case Brian has updated thoughts on this issue as he's the original author.

Thanks.

**Comment by josephsl** [comment]
Hi,

2025 update: since we haven't heard from Brian G, I think we should either close this or continue with current comments (from 2017) to help us figure out what to do.

Thanks.

**Comment by Adriani90** [comment]
Given we have already open issues regarding pitch settings for beeps etc. I am closing this issue as suggested above and also in favor of #10116, #10886 and #9308.
